compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/context/goto_ctx.rs

```diff
@@ -54,13 +57,15 @@ pub struct GotocCtx<'tcx> {
 impl<'tcx> GotocCtx<'tcx> {
     pub fn new(tcx: TyCtxt<'tcx>) -> GotocCtx<'tcx> {
         let (thks, fhks) = type_and_fn_hooks();
+        let crate_name = full_crate_name(tcx);
```

Comment:
Why not just call this `full_crate_name`

Reply:
Changed.